README.rst

```diff
@@ -224,6 +224,32 @@ has a value which is not in the expected range
 Example: ping is by default a critical metric which is expected to be always 1
 (reachable).
 
+``Available Checks``
+--------------------
+
+``Ping``
+~~~~~~~~
+
+This check returns information on device ``uptime``, ``downtime``,
+``RTT (Round trip time) maximum, minimum and average``.
+It creates a ``Ping`` metric to store this information and an ``AlertSettings``
+to monitor them.
+The Charts ``uptime``, ``packet loss`` and ``rtt`` are created. The ``fping``
+command is used to collect these metrics.
+You may choose to disable auto creation of this check by using the setting
```

Comment:
Instead of `by using the setting ...` I'd say `by setting ... to False` which is more concrete